daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json

```diff
@@ -0,0 +1,46 @@
+{
+  "Name": "build-debian-11-worker-arm64",
+  "Project": "gce-image-builder",
+  "Zone": "us-central1-a",
+  "GCSPath": "gs://gce-image-build-bucket/daisy/${USERNAME}",
+  "Vars": {
+    "build_date": {
+      "Value": "${DATE}w",
```

Comment:
extra w?

Reply:
it exists in debian-10-worker. I thought it's by purpose. Let me remove both then.

Reply:
Oooh it does. It is to differentiate the worker image from the normal one. Ok yeah lets leave it.